packages/llm/README.md

```diff
@@ -1 +1,153 @@
-# @synstack/json
+# @synstack/llm

```

Comment:
This one should be split accross multiple markdown files
- Tool calling
- Message templating
- Completion
- Runners